clients/vscode/perl/README.md

```diff
@@ -18,20 +18,20 @@ Language Server and Debugger for Perl
   * Support for coro threads
   * Breakpoints 
   * Conditional breakpoints
-  * Breakpoints can be set while programm runs and for modules not yet loaded
+  * Breakpoints can be set while program runs and for modules not yet loaded
   * Variable view, can switch to every stack frame or coro thread
   * Set variable
   * Watch variable
   * Tooltips with variable values
-  * Evaluate perl code in debuggee, in context of every stack frame of coro thread
```

Comment:
The perl code is really evaluated in the debuggee not the debugger. The debuggee is the program that is actualy debugged

Reply:
Oh! Thanks for clarifying. I pushed a commit that reverts it back to "debuggee"